docs/guidance/community_engagement.md

```diff
@@ -25,6 +27,19 @@ Your community needs to be a safe, inclusive, and inviting space for everyone to
 - help keep the community platform well organized to make it easy for people to use, for example, by maintaining a clear list of discussion categories and provide guidance on how to post
 - promote engagement on the community platform by encouraging people to post or by posting what they have heard from people
 
+Before openining a community, the community manager should ensure that there is content planning covering at least the initial month of community's activity.
+
+### Evaluating your community
+
+It is important to set metrics and indicators to evaluate your community's success over time. 
+
+Metrics and indicators can refer to the purpose why the community has been built, and to how healthy and alive the community is. In both cases, an ongoing evaluation of your community is a useful exercise to understand whether there is a need to maintain the community, whether the invested resources are appropriate, and whether the users are getting a significant value from participating.
+
+Metrics can include quantitative data generated by the system, such as pages visited, time spent over articles, number of users, active users, new posts etc. For complex Organizations such as WHO, however, it is recommended that the evaluation is also complemented by qualitative data and feedback. This will include users interviews, surveys, discussions, and qualitative analysys of trends.
```

Comment:
```suggestion
Metrics can include quantitative data generated by the system, such as pages visited, time spent over articles, number of users, active users, new posts etc. For complex Organizations such as WHO, however, it is recommended that the evaluation is also complemented by qualitative data and feedback. This will include user interviews, surveys, discussions, and qualitative analyses of trends.
```